tests/unit-tests/test_config_postfix_formatting.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+"""
+:copyright: Copyright 2020-2022 Sphinx Confluence Builder Contributors (AUTHORS)
+:license: BSD-2-Clause (LICENSE)
+"""
+
+import os
+from sphinxcontrib.confluencebuilder.exceptions import \
+    ConfluenceConfigurationError
+from sphinxcontrib.confluencebuilder.state import ConfluenceState
+from tests.lib.testcase import ConfluenceTestCase
+from tests.lib import parse
+from tests.lib.testcase import setup_builder
+from unittest.mock import patch
```

Comment:
Note that we still support Python 2.7, so the mock module will not be available.

Reply:
I added the requirement for mock on python < 3.3. To keep the tests clean. If you would perfer not to use mock for this repos due to the python <3.3 requirement I can alter these tests as well to just use a known hash for the docname its not a big deal for me either way

Reply:
👍, I have no problem introducing a `mock` requirement for older Python versions for the test environment.